clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -238,23 +235,10 @@ const ConnectorParametersForm: React.FC<ConnectorParametersFormProps> = ({
   };
 
   const handleTestConnectionClick = async () => {
-    try {
```

Comment:
woof okay I think I may have given you not great advice removing this `unwrap`. the case that this breaks is if the fetch itself fails, for example if the backend server is down, then an error will be thrown and nothing catches it. (you can test this by getting your admin UI to this screen so it's ready to test the connection, then shutting your backend down, then clicking "Test connection"). adding the try catch back doesn't fix things, since I'm guessing the `unwrap` is forcing the error to throw. but I don't want to add the `unwrap` back since then we lose `fulfilledTimeStamp` 😐 

it seems to originate in the [redux slice here](https://github.com/ethyca/fides/blob/3601-test-connection-toasts/clients/admin-ui/src/features/datastore-connections/datastore-connection.slice.ts#L311). all of that logic under `onQueryStarted` is very unusual IMO (most of our other slice files when defining endpoints are much simpler) and should probably be refactored. I think we can remove the `try catch` inside `datastore-connection.slice` and then that'll let the error bubble up to the toast to be handled. but it might mess up `ConnectionGrid` which also uses that query.

it might be that this is too complicated to roll into this ticket, in which case let's make a follow up issue for it 👍